app/components/post_draft/send_handler/send_handler.test.tsx

```diff
@@ -135,9 +150,13 @@ describe('components/post_draft/send_handler/SendHandler', () => {
         expect(sendDraftButton).toBeTruthy();
         expect(wrapper.getByText('Send draft')).toBeTruthy();
 
-        // Verify the button is enabled when there's a message (should be pressable)
+        // Manually trigger the button press
         fireEvent.press(sendDraftButton);
-        await waitFor(() => expect(sendMessageWithAlert).toHaveBeenCalled());
+
+        // Use a simple timeout instead of waitFor
```

Comment:
(For my own knowledge) Why `waitFor` doesn't work here?

Reply:
Ah, this was missed. `WaitFor` work here.

Reply:
ok so you'll be changing it back, correct?